embed/oko_attached/src/components/attached_initialized/color_scheme.ts

```diff
@@ -8,7 +8,12 @@ export function setColorScheme(theme: Theme) {
   // Set color-scheme to support both light and dark
   // Browser will automatically match parent dApp's color-scheme
   // This prevents browser from adding opaque background
-  root.style.colorScheme = "light dark";
+
+  // NOTE - Applying a color-scheme to the body of an iframe causes a bug where,
+  // if the host site doesn't have a color-scheme and the browser's default is dark,
+  // the background changes to black.
+  // Therefore, we force a light mode in the iframe(in sdk_core not here) @retto
+  // root.style.colorScheme = "light dark";
```

Comment:
It might be better to put that comment in `sdk/oko_sdk_core/src/iframe/index.ts`. I’ll probably forget about it myself in three months.